authors/apps/articles/views.py

```diff
@@ -625,3 +629,19 @@ def delete(self, request, pk, format=None):
         self.check_object_permissions(request, report)
         report.delete()
         return Response(status=status.HTTP_200_OK)
+
+
+class CreateListAuthorsAPIView(ListAPIView):
+    """
+    Allow any authenticated/verified users to hit this endpoint.
+    List all authors plus their profiles.
+    """
+    permission_classes = (IsAuthenticated, IsVerified,)
+    serializer_class = UserSerializer
+    queryset = User.objects.all()
+    
+    def get_queryset(self):
+        queryset = Articles.objects.all()
+        articles =[article.author.id for article in queryset]
```

Comment:
Just a suggestion, you could make use of `values_list` when querying to return a list of `author ids` so you don't have to create a list comprehension

Reply:
done and done